Presentation.Web/app/helpers/select2-option-format-helper.ts

```diff
@@ -12,12 +12,35 @@
             return Select2OptionsFormatHelper.formatText(org.text, org.optionalObjectContext?.cvrNumber);
         }
 
+        public static addIndentationToUnitChildren(orgUnit: Kitos.Models.Api.Organization.OrganizationUnit, indentationLevel: number): Kitos.Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<number>[] {
+            const options: Kitos.Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<number>[] = [];
+            Select2OptionsFormatHelper.visitUnit(orgUnit, 0, options);
+
+            return options;
+        }
+
         private static formatText(text: string, subText?: string): string {
             let result = `<div>${text}</div>`;
             if (subText) {
                 result += `<div class="small">${subText}</div>`;
             }
             return result;
         }
+        
+        private static visitUnit(orgUnit: Kitos.Models.Api.Organization.OrganizationUnit, indentationLevel: number, options: Kitos.Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<number> []) {
+            const option = {
+                id: String(orgUnit.id),
+                uuid: orgUnit.uuid,
+                text: orgUnit.name,
+                indentationLevel: indentationLevel
+            };
+
+            options.push(option);
+
+            _.each(orgUnit.children, function (child) {
```

Comment:
why use lodash when you can use the forEach build-in function in TS? (you do that in another helper)